### PR TITLE
Feature listar perfiles 160031998

### PR DESCRIPTION
**Pull request by franbe7**
Esta es la rama correspondiente a listar perfiles. 
Se hicieron pruebas de humo para cumplir los criterios de aceptación.

### CODE REVIEW

File: src/components/ProfileList.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                  id={user.id}
+                  image={user.image}
+                  last_name={user.last_name}
+                  first_name={user.first_name}
+                />
+              )),
+            )}
+          </ul>
+        ) : (
+          <div className="alert alert-info message">There are no users</div>
```
Comment (manuel-saravia) [inline src/components/ProfileList.jsx]: The message should be in spanish, shouldn't it?

Comment (franbe7) [reply src/components/ProfileList.jsx]: The message was changed

**Comment by manuel-saravia** [comment]
PRA!

File: src/components/ProfileItem.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+              className="profile-left__image"
+              alt="logo"
+            />
+          </div>
+        </div>
+        <div className="media-body profile-right">
+          <p className="profile-right__lastname">
+            {lastName}
+,
+            {firstName}
```
Comment (luchong12) [inline src/components/ProfileItem.jsx]: Is there a reason why this isn't in just one line and with a space in between, like this: `{lastName}, {firstName}` ?

**Pull request merged**